### PR TITLE
[POL-WIP] History of osu!

### PR DESCRIPTION
**Pull request by vistafan12**
still unfinished

### CODE REVIEW

File: wiki/History-of-osu!/PL/1.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# 2007
```
Comment (TPGPL) [inline wiki/History-of-osu!/PL/1.md]: use "##" header instead of "#"

File: wiki/History-of-osu!/PL/1.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+![ouentest](http://vistafan12.ct8.pl/i/osu/Osu2007.jpg)
+### Lipiec
+osu! rozpoczął swoją przygodę jako proof-of-concept (dowód na koncept) o nazwie "ouentest". Pierwsza wersja została wydana 1 lipca 2007 roku dla małej grupy odbiorców (głównie to byli przyjaciele). Gra zawierała bardzo podstawowy tryb edycji i dzięki niemu można było stworzyć prostą bitmapę. 
+Nie było żadnych sliderów oraz spinnerów, ale liczniki punktacji / combo były obecne w grze. Kodowanie pierwszej wersji zajeło 16 godzin. pUpdater został wporwadzony (obecnie znany jako osume) do obsługi aktualizacji do najnowszej wersji.
```
Comment (TPGPL) [inline wiki/History-of-osu!/PL/1.md]: fix grammar and fast-keyboard mistakes

File: wiki/History-of-osu!/PL/1.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+Pod koniec lipca, menu gry zostało zmienionione, aby dodać trochę "stylu". Początkowe wdrożenie sliderów zostały zakończone, z pomarańczową piłeczką na czele (która była bardzo brzydka). 
+![Tak wyglądał Gameplay](http://vistafan12.ct8.pl/i/osu/200707-08.jpg)
```
Comment (TPGPL) [inline wiki/History-of-osu!/PL/1.md]: Add hover text to images `Alt Text`

File: wiki/History-of-osu!/PL/1.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+### Tak wyglądał Gameplay
```
Comment (TPGPL) [inline wiki/History-of-osu!/PL/1.md]: Delete this header, the hover text will do it

File: wiki/History-of-osu!/PL/1.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+### Sierpień
+Menu główne zostało ponownie zaaktualizowane, a edytor dostał wiele nowych funkcji (w tym wsparcie dla kopiowania / wklejania). Windowsowy pasek menu oraz kursor myszy zostały dodane, co ułatwiało prace nad bitmapą.
+Lokalne wysokie wyniki teraz są o wiele lepsze. Format pliku .osu został zmieniony i całkowicie rozbudowany, obejmuję konwersję starszych plików do nowego formatu.
+![Menu wyboru piosenki](http://vistafan12.ct8.pl/i/osu/200708-01.jpg)
+### Menu wyboru piosenki
```
Comment (TPGPL) [inline wiki/History-of-osu!/PL/1.md]: Delete this header, the hover text will do it

File: wiki/History-of-osu!/PL/1.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+![Tak wyglądał Gameplay](http://vistafan12.ct8.pl/i/osu/200708-04.jpg)
+### Tak wyglądał Gameplay
```
Comment (TPGPL) [inline wiki/History-of-osu!/PL/1.md]: Delete this header, the hover text will do it

File: wiki/History-of-osu!/PL/1.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+### Wrzesień
+![Publiczna beta!](http://vistafan12.ct8.pl/i/osu/200709-Release.png)
+A oto wielkopomna chwila. Osu! wreszcie ujżało światło dzienne. Strona osu została postawiona na domowym serwerze peppego (ppy.sh).
```
Comment (TPGPL) [inline wiki/History-of-osu!/PL/1.md]: Even if it's beginning of the sentence, **don't capitalise osu!**

File: wiki/History-of-osu!/PL/1.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+### Sierpień
+Menu główne zostało ponownie zaaktualizowane, a edytor dostał wiele nowych funkcji (w tym wsparcie dla kopiowania / wklejania). Windowsowy pasek menu oraz kursor myszy zostały dodane, co ułatwiało prace nad bitmapą.
+Lokalne wysokie wyniki teraz są o wiele lepsze. Format pliku .osu został zmieniony i całkowicie rozbudowany, obejmuję konwersję starszych plików do nowego formatu.
+![Menu wyboru piosenki](http://vistafan12.ct8.pl/i/osu/200708-01.jpg)
+### Menu wyboru piosenki
+![Tak wyglądał Gameplay](http://vistafan12.ct8.pl/i/osu/200708-04.jpg)
+### Tak wyglądał Gameplay
+
+### Wrzesień
+![Publiczna beta!](http://vistafan12.ct8.pl/i/osu/200709-Release.png)
```
Comment (TPGPL) [inline wiki/History-of-osu!/PL/1.md]: Add hover text

File: wiki/History-of-osu!/PL/1.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+Nie było żadnych sliderów oraz spinnerów, ale liczniki punktacji / combo były obecne w grze. Kodowanie pierwszej wersji zajeło 16 godzin. pUpdater został wporwadzony (obecnie znany jako osume) do obsługi aktualizacji do najnowszej wersji.
+Pod koniec lipca, menu gry zostało zmienionione, aby dodać trochę "stylu". Początkowe wdrożenie sliderów zostały zakończone, z pomarańczową piłeczką na czele (która była bardzo brzydka). 
+![Tak wyglądał Gameplay](http://vistafan12.ct8.pl/i/osu/200707-08.jpg)
+### Tak wyglądał Gameplay
+
+### Sierpień
+Menu główne zostało ponownie zaaktualizowane, a edytor dostał wiele nowych funkcji (w tym wsparcie dla kopiowania / wklejania). Windowsowy pasek menu oraz kursor myszy zostały dodane, co ułatwiało prace nad bitmapą.
+Lokalne wysokie wyniki teraz są o wiele lepsze. Format pliku .osu został zmieniony i całkowicie rozbudowany, obejmuję konwersję starszych plików do nowego formatu.
+![Menu wyboru piosenki](http://vistafan12.ct8.pl/i/osu/200708-01.jpg)
```
Comment (TPGPL) [inline wiki/History-of-osu!/PL/1.md]: Add hover text

**Comment by Repflez** [comment]
Is this finished?

**Comment by vistafan12** [comment]
Repflez: Nope.

**Comment by vistafan12** [comment]
Ok.

File: wiki/History-of-osu!/PL/1.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
+## 2007
+![ouentest](http://vistafan12.pl/i/osu/Osu2007.jpg "ouentest")
```
Comment (RaikireHiuduo) [inline wiki/History-of-osu!/PL/1.md]: `ouentest`

File: wiki/History-of-osu!/PL/1.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+Mała gierka zdobyła popularność dzięki stronie bemanistyle.com która przyciągnęła sporo zainteresowanych osób z wcześniejszym doświadczeniem rytmicznej gry.
+Zostały dodane pierwsze mody - Easy oraz No Fail. Duża liczba zmian przyciągała uwagę.
+## Październik
+1 październik był dniem, kiedy rankingi na forum zostały udostępnione publicznie. Wtedy również zapoczątkowano sposób pakowania pliku .osz oraz bazę beatmap online z bardzo wczesnym systemem składania wniosków. Rankingi online zostały dodane do interfejsu gry, a także były wyświetlane na stronie internetowej.
+###
+![Raport](./images/200710-01.jpg "Raport")
+###
+Ekran wyboru piosenki został ulepszony, dodano grupowanie trudności oraz bardziej intuicyjny ruch koła piosenki. Dodano również kontrolę kółka na klawiaturze. Gracze mogli tworzyć skórki oraz edytować skórkę bitmapy. Dodano również wsparcie dla TabletPC.
+###
+![Wybieranie piosenek](http://vistafan12.pl/i/osu/200710-03.jpg "Wybieranie piosenek")
```
Comment (RaikireHiuduo) [inline wiki/History-of-osu!/PL/1.md]: `Wybieranie piosenek`

File: wiki/History-of-osu!/PL/1.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
+## 2007
```
Comment (RaikireHiuduo) [inline wiki/History-of-osu!/PL/1.md]: Add this above the `## 2007`:-

`# Historia osu!`

Comment (vistafan12) [inline wiki/History-of-osu!/PL/1.md]: Ok.

**Comment by vistafan12** [comment]
check this ^

**Comment by Repflez** [comment]
So, this is ok to merge now?

**Comment by vistafan12** [comment]
yes

**Pull request merged**